src/lib/RpcApi.ts

```diff
@@ -3,7 +3,9 @@ import { BrowserDetection } from '@nimiq/utils';
 import { RootState } from '@/store';
 import { Store } from 'vuex';
 import Router from 'vue-router';
-import { AccountsRequest, RequestType, RpcRequest, RpcResult } from '@/lib/RequestTypes';
+import { RequestType } from './RequestTypes';
+import { AccountsRequest } from './AccountsRequest';
+import { RpcRequest, RpcResult } from './PublicRequestTypes';
```

Comment:
I _really_ dont like that the request types are now split between two files, public and internal. Is that really necessary? What benefits does it bring?

Reply:
It clearly separates the types which are relevant for users of the client from the ones which are just internally used.